tests/test_to_container.py

```diff
@@ -230,22 +227,6 @@ def test_string_interpolation_with_readonly_parent() -> None:
     }
 
 
-@mark.parametrize(
-    "src,expected",
-    [
-        param(DictConfig(content="${bar}"), "${bar}", id="DictConfig"),
-        param(
-            OmegaConf.create({"foo": DictConfig(content="${bar}")}),
-            {"foo": "${bar}"},
-            id="nested_DictConfig",
-        ),
-    ],
-)
-def test_to_container_missing_inter_no_resolve(src: Any, expected: Any) -> None:
```

Comment:
Why is this test being removed?

Reply:
My thinking was that this is redundant with the new `TestResolveBadInterpolation.test_no_resolve` method below.
However, it looks like I missed one of the test cases (the case of `DictConfig(content="${bar}")`). I'll make sure to add it back in, either by restoring this test or by adding to the `TestResolveBadInterpolation` class below.

Reply:
Done in https://github.com/omry/omegaconf/pull/730/commits/05221aa1235b298ae6bd5232af9f60bf27bd2ac7.

Reply:
I may be missing something, but I don't see the test case with just `DictConfig(content="${bar}"))` in this commit (?)

Reply:
Nope, you're absolutely right. Thanks for keeping an eye on the details...

Reply:
Done in https://github.com/omry/omegaconf/pull/730/commits/a059a8fc07b733ff3c52b413b0772009ed8c772a.